NEWS.md

```diff
@@ -32,6 +32,13 @@ Language changes
   * `{ }` expressions now use `braces` and `bracescat` as expression heads instead
     of `cell1d` and `cell2d`, and parse similarly to `vect` and `vcat` ([#8470]).
 
+  * The `global` keyword now always introduces a new binding when appearing in toplevel (global) scope.
+    Whereas, previously, embedding it in a block (such as `begin; global sin; nothing; end`) would change its meaning.
+    Additionally, the new bindings are now created before the statement is executed.
+    For example, `f() = (global sin = rand(); nothing)` will now reliably shadow the `Base.sin` function,
```

Comment:
https://en.wikipedia.org/wiki/Relative_clause#Restrictive_and_non-restrictive

If it was a complete fragment, this would be a comma splice. The rest of the sentence, on the other hand, is complete and stands on its own.